test/twine_test.coffee

```diff
@@ -350,16 +348,17 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       Twine.shouldDiscardEvent.click = -> true
 
-      jQuery(node).click()
+      triggerEvent node, "click"
```

Comment:
There's instances of `triggerEvent(node, "click")` here too (with parenthesis), should they be consistent?